src/components/targets-map/TargetsMap.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Map, TileLayer } from "react-leaflet";
+
+import { mapConstants } from "../../constants/map.constants";
+function TargetsMap() { 
```

Comment:
```suggestion

const TargetsMap = () => { 
```

Reply:
Why is this better?

Reply:
This is ES6 nomenclature versus ES5. Both are OK but the second one, Tami's, is preferred

Reply:
I found a [great link](https://stackoverflow.com/a/34361380/12488350) explaining the difference between them.